src/components/builds/BuildCard.tsx

```diff
@@ -78,16 +77,33 @@ export default function BuildCard(props: Props) {
   );
 
   let classes = useStyles();
-  let theme = useTheme();
   const navigate = useNavigate();
+  const isDrawerOpen = useRecoilValue(cirrusOpenDrawerState);
 
   const pageWidth = usePageWidth();
 
-  let isMdScreenWidth = pageWidth >= theme.breakpoints.values.md;
+  // For default Chip component values
+  let themeOptions = useRecoilValue(muiThemeOptions);
+
+  if (isDrawerOpen) {
+    themeOptions = {
+      ...themeOptions,
+      breakpoints: {
+        values: {
+          xs: 0,
+          sm: 900,
+          md: 1200,
+          lg: 1600,
+          xl: 1800,
```

Comment:
How did you end up with this numbers? They seems to be larger than the [default one](https://mui.com/material-ui/customization/breakpoints/#default-breakpoints). My understanding that it should be smaller so the breakpoints "triggered" on smaller displays.

Reply:
New breakpoints allow me to implement styles for xs on screens 0-900px, instead 0-600px.